pkg/chrootarchive/archive.go

```diff
@@ -7,16 +7,21 @@ import (
 	"os"
 	"os/user"
 	"path/filepath"
+	"runtime"
 
 	"github.com/docker/docker/pkg/archive"
 	"github.com/docker/docker/pkg/idtools"
 )
 
 func init() {
-	// initialize nss libraries in Glibc so that the dynamic libraries are loaded in the host
-	// environment not in the chroot from untrusted files.
-	_, _ = user.Lookup("docker")
-	_, _ = net.LookupHost("localhost")
+	// Any go binary that uses this package will panic on Windows Nano server due to missing
+	// netapi32.dll, which is required by user.Lookup().
+	if runtime.GOOS == "windows" {
```

Comment:
Guess this was meant to be

```suggestion
	if runtime.GOOS != "windows" {
```

Reply:
indeed. I need sleep...

Thanks for catching that!